.changeset/hot-items-smoke.md

```diff
@@ -0,0 +1,63 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+---
+
+**BREAKING** Allow refreshing Kubernetes clusters
```

Comment:
I assume this is only breaking if you implement your own custom supplier right?

Reply:
Yes, only then, should I add that to the "breaking" line?  E.g. like this:

> **BREAKING** Custom cluster suppliers need to cache their `getClusters` result
>
> To allow refreshing Kubernetes clusters [...]

Reply:
Yeah I think that's a better start! 👍

Reply:
Changed this as described above!